README.md

```diff
@@ -1,3 +1,15 @@
+## Pre-requisites
+
+**Hook**
+ - Python 2 or 3 with pip
+ 
+**Registration script**
+ - Python 3
+
+The pre-commit framework is installed by pip, the python package manager. To install it on your machine either brew install it or get it from your distributions package manager (it might be called `python3-pip`, or `pip3`)
+
+Currently, our registration script's github oauth flow only works with OTP keys (such as ubikeys), not TOTP keys. If you use such a key, please skip registration.
```

Comment:
This text is not correct. We do support TOTP like Google Authenticator / Authy. We don't support YubiKey's Fido U2F (https://www.yubico.com/authentication-standards/fido-u2f/)